examples/quickstart/jupyter-notebooks/kafka-tutorial.ipynb

```diff
@@ -0,0 +1,753 @@
+      "text/html": [
+       "<div class=\"druid\"><table>\n",
+       "<tr><th>__time</th><th>username</th><th>post_title</th><th>views</th><th>upvotes</th><th>comments</th><th>edited</th></tr>\n",
+       "<tr><td>2023-01-03T16:40:21.501Z</td><td>willow_bean</td><td>This title is required</td><td>15284</td><td>124</td><td>21</td><td>True</td></tr>\n",
+       "<tr><td>2023-03-15T23:59:46.920Z</td><td>mish_middy</td><td>l0g!9IQdgd9uOBoBVAr.eNBxxDCrn_</td><td>11674</td><td>53</td><td>14</td><td>True</td></tr>\n",
+       "<tr><td>2023-03-15T23:59:46.920Z</td><td>rocket</td><td>rpF!DoCRz:Ityskmor752VZIPOPb7_gjbNNUCxmRDMhaIJlWJBKucW</td><td>2791</td><td>56</td><td>1</td><td>True</td></tr>\n",
+       "<tr><td>2023-03-15T23:59:46.921Z</td><td>jojo</td><td>pb3gqohSiTRfy.CBU:n;SNHMHH5WG._2Am,L1Z!eS7zg:&#x27;E3rAsbRhbOh3Px!8fVFhKTHx,&#x27;cvUbHmzT,r,1d3</td><td>2022</td><td>47</td><td>9</td><td>False</td></tr>\n",
+       "<tr><td>2023-03-15T23:59:46.921Z</td><td>willow_bean</td><td>06!JKoyFKhpuHvZtXgB4GpYjcG9GGn5</td><td>36305</td><td>78</td><td>-2</td><td>True</td></tr>\n",
+       "</table></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sql = '''\n",
+    "SELECT * FROM social_media LIMIT 5\n",
+    "'''\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this social media scenario, each incoming event represents a post on social media, for which you collect the timestamp, username, and post metadata. You are interested in analyzing the total number of upvotes for all posts, compared between users. Preview this data with the following query:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"druid\"><table>\n",
+       "<tr><th>num_posts</th><th>total_upvotes</th><th>username</th></tr>\n",
+       "<tr><td>193219</td><td>13406871</td><td>jambalion</td></tr>\n",
+       "<tr><td>193495</td><td>13446847</td><td>miette</td></tr>\n",
+       "<tr><td>193608</td><td>13452801</td><td>milton</td></tr>\n",
+       "<tr><td>193720</td><td>13467209</td><td>willow_bean</td></tr>\n",
+       "<tr><td>193851</td><td>13461992</td><td>mish_middy</td></tr>\n",
+       "<tr><td>193908</td><td>13480426</td><td>shadow</td></tr>\n",
+       "<tr><td>193949</td><td>13480342</td><td>jojo</td></tr>\n",
+       "<tr><td>194197</td><td>13492537</td><td>rocket</td></tr>\n",
+       "</table></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sql = '''\n",
+    "SELECT COUNT(post_title) as num_posts, SUM(upvotes) as total_upvotes, username FROM social_media GROUP BY username ORDER BY num_posts\n",
```

Comment:
since this is multiline, can we actually break at the commas and keywords:
```
SELECT COUNT(post_title) as num_posts,
SUM(upvotes),
username
FROM social_media...
```

Reply:
Updated